Automattic/jetpack
# Instant Search: Add the "Clear filters" button to the tab loop.

#### Changes proposed in this Pull Request:


The "Clear filters" button is now a `button` instead of an `a`. This allows it to participate in the tab loop in Firefox and no longer breaks the loop in other browsers.

**Note:** Just a standard button now. The old styling wasn't that nice but we can tweak that based on feedback.

Before:
![ClearOld](https://user-images.githubusercontent.com/40267301/181228535-0d4b4e66-7220-4cfa-8764-62ddb34d734a.png)

After:
![ClearNew](https://user-images.githubusercontent.com/40267301/181228563-eee38497-a240-45c9-92da-976b3c5b892a.png)

#### Other information:


#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Open the Instant Search overlay.
2. Click on a filter or two so the "Clear filters" button appears.
3. Try tabbing to it.
4. Confirm that both Return and Space activate the button and clear the filters.
